sqlitecluster/SQLiteNode.h

```diff
@@ -364,9 +364,12 @@ class SQLiteNode : public STCPManager {
     // Stopwatch to track if we're giving up on the server preventing a standdown.
     SStopwatch _standDownTimeout;
 
-   // Our current State.
+    // Our current State.
     atomic<SQLiteNodeState> _state;
 
+    // Keeps track if we have closed the command port for commits fallen behind.
+    bool _blockedCommandPort{false};
```

Comment:
Since we block the command port for other reasons, we should put the fact that this is being set based on the node being behind on the variable name, for clarity
```suggestion
    bool _blockedCommandPortForBeingBehind{false};
```

But do we really need this variable? Is there an easy way to check the reason the command port is blocked so we can do something like 
```cpp
_blockedCommandPortForBeingBehind = getReason() == "COMMITS_LAGGING_BEHIND";
if (peer == _leadPeer && currentCommitDifference >= 12'500 && !_blockedCommandPort) {
    ...
```

Reply:
> But do we really need this variable? Is there an easy way to check the reason the command port is blocked so we can do something like

tl;dr: Adding this as a separate variable so we don't need to add complexity to the current methods related to block reasons. We had an implementation like you're suggesting before, but that would require adding new locks/changing current lock types.

This was discussed in this thread: https://expensify.slack.com/archives/CC7NECV4L/p1734981555759429?thread_ts=1733270912.693829&cid=CC7NECV4L